souschef.py

```diff
@@ -16,25 +34,31 @@
 PATH = path_builder.PathBuilder(channel_name=CHANNEL_NAME)  # Keeps track of path to write to csv
 WRITE_TO_PATH = "{}{}{}.zip".format(os.path.dirname(os.path.realpath(__file__)), os.path.sep, CHANNEL_NAME) # Where to generate zip file
 
+# webcache
+###############################################################
+sess = requests.Session()
+#cache = FileCache('.webcache')
```

Comment:
This is good for testing, but in the long run should enable some sort of webcache to avoid re-downloading resources every time chef runs.